streampipes-rest/src/main/java/org/apache/streampipes/rest/impl/Version.java

```diff
@@ -33,6 +34,7 @@ public class Version extends AbstractAuthGuardedRestResource {
   @GET
   @Path("/versions")
```

Comment:
I would propose to rename the endpoint to `check`, `health-check` or `status`.
Any other ideas?

Reply:
@tenthe @dominikriemer

Reply:
We could rename this to Status and add more information on registered extension services and other status info.
What info do we want to return here?